### ISSUE TITLE
TODO: Remove items not in 0.5.3

### ISSUE THREAD

**Issue by Daribon** [The-Alpha-Project/alpha-core#317]
There are vast amount of item entries at high entry that should not exist in 0.5.3 alpha.

![image](https://user-images.githubusercontent.com/6137576/179373635-11772246-f39a-40cd-9da5-79442bd410d9.png)
Image from item_template DB.
No such spells exist spell.dbc in 0.5.3 client.
Instant poison rank 2 exists in the 0.5.5 spell.dbc.

![image](https://user-images.githubusercontent.com/6137576/179373645-e5dd9aeb-1d35-4564-818c-7e902a231145.png)
Starting gear exists in 0.5.3 client.
Which gives us us a baseline that 0.5.3 ends somewhere between entry 6,1k and 7k.

![image](https://user-images.githubusercontent.com/6137576/179373671-f61f0c74-822f-4030-979d-b968f738efb1.png)
None of those spells exists in 0.5.3 client.

![image](https://user-images.githubusercontent.com/6137576/179373678-450fd0ce-6261-451a-b0cf-824ac8fbdad6.png)
![image](https://user-images.githubusercontent.com/6137576/179373679-640eb0ab-c788-41c6-9e03-f0c283703cd3.png)
All of the spells above exists in the 0.5.3 client.
Therefor 0.5.3 ends somewhere between entry 6,1k and 6,4k.

![image](https://user-images.githubusercontent.com/6137576/179373704-0b23fa7f-3a85-4415-9706-812a3f1f7c09.png)
Last item Pattern: Dark silk shirt in the 0.5.3 that has a spell that exists in the 0.5.3 client.
Meaning that 0.5.3 ends somewhere around 6,4k entry.

![image](https://user-images.githubusercontent.com/6137576/179373808-4da660a9-1a51-4e80-b71a-c2feba313d0b.png)
If one takes last item displayid from itemdisplayinfo.dbc into account.
Then last item is the quest item Shadowstalker Scalp.
It is an unique item displayid and only used for that item.

![image](https://user-images.githubusercontent.com/6137576/179373832-1aecd359-3e0d-4c4b-b359-817ecb0a368f.png)
The item drops off Hatefury Shadowstalker, a NPC that does exist in the 0.5.3 client.

TL;DR last item in 0.5.3 should be item entry 6441.

**Describe what needs to be implemented or enhanced**
Remove all items above item entry 6441.

**Expected end result**
Only items that existed in 0.5.3 will be available.

**Guidance**
Nuke all items entries above 6441 from item_template, npc_vendor and item_loot_template.
